pkg/cmd/wheredoresourcescomefrom/areyoumymother.go

```diff
@@ -0,0 +1,23 @@
+package wheredoresourcescomefrom
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/reactionchain"
+	"github.com/spf13/cobra"
+)
+
+func NewResourceChainCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "where-do-resources-come-from",
```

Comment:
not sure I understand where this is going. I expected just some simple data structure with namespaces and names that we can refer to from the controllers.

This code here does not really serve the purpose of documentation.

Reply:
> not sure I understand where this is going. I expected just some simple data structure with namespaces and names that we can refer to from the controllers.
> 
> This code here does not really serve the purpose of documentation.

Payload leads to CVO leads to aggregator cert rotation controller leads to configmaps/managed-aggregator-client-ca, separately the installer leads to configmaps/initial-aggregator-client-ca, configmaps/initial-aggregator-client-ca and configmaps/managed-aggregator-client-ca are combined in configmaps/aggregator-client-ca which is then copied to configmaps/kube-apiserver-aggregator-client-ca

Pretty sure this is the smallest way to express that.